packages/design-picker/src/utils/available-designs.ts

```diff
@@ -104,5 +104,24 @@ export function getAvailableDesigns( {
 		}
 	}
 
+	// Force blank canvas design to always be first in the list
+	const blankCanvasIndex = designs.featured.findIndex( ( design ) =>
+		isBlankCanvasDesign( design )
+	);
+	if ( blankCanvasIndex > -1 ) {
+		designs = {
+			...designs,
+			featured: [
+				designs.featured[ blankCanvasIndex ],
+				...designs.featured.slice( 0, blankCanvasIndex ),
+				...designs.featured.slice( blankCanvasIndex + 1 ),
+			],
+		};
+	}
```

Comment:
Optional: I'd probably do this. I also don't see harm in mutating the same array. But kept the slicing anyways if you like it.

```suggestion
	designs.featured = designs.featured
		.slice( 0 )
		.sort( ( a, b ) => Number( isBlankCanvasDesign( b ) ) - Number( isBlankCanvasDesign( a ) ) );
```

Reply:
No need for slicing — done in https://github.com/Automattic/wp-calypso/pull/52293/commits/a58ce8cc198680789edf95908dafa2772f72021f